doc/atcommands.txt

```diff
@@ -1267,31 +1267,47 @@ Example:
 
 ---------------------------------------
 
-@unloadnpc <npc name>
-
-Unloads an NPC.
+@unloadnpc <NPC_name> {<flag>}
+
+Unloads a NPC.
+Flag:
+	0 - do not unload mobs spawned by NPCs in file
+	1 - unload mobs spawned by NPCs in file
```

Comment:
maybe mention here too about commands?
```suggestion
	1 - unload mobs spawned by NPCs and atcommands
```

Reply:
In my opinion this would be confusing, because it's really just _atcommand(@monster ...)_ used inside a script. Atcommands used by players are not affected.